tartufo/util.py

```diff
@@ -36,6 +37,15 @@ def del_rw(_func: Callable, name: str, _exc: Exception) -> None:
     os.remove(name)
 
 
+def convert_regexes_to_rules(regexes: Dict[str, Pattern]) -> Dict[str, Rule]:
+    rules: Dict[str, Rule] = {}
+    for regex_name in regexes:
+        rules[regex_name] = Rule(
+            name=regex_name, pattern=regexes[regex_name], path_pattern=None
+        )
+    return rules
```

Comment:
Also not a necessary change here, but another common Python pattern that could be used in this instance: [dictionary comprehensions](https://www.python.org/dev/peps/pep-0274/)!

```python
return {name: Rule(name=name, pattern=pattern, path_pattern=None) for name, pattern in regexes.items()}
```

6 lines down to 1 (as long as `black` keeps it that way...), and temporary variables eliminated!

Reply:
Given the urgency of this PR, do you think we have time for changes like this? Seems more like a personal preference.

Reply:
I don't think this needs to be a blocker on the PR. It is something worth discussing and settling on an agreed pattern for use in the project moving forward.

Reply:
Like Mark and I said, this is not a necessary change or a blocker. It's just a suggestion to show a pattern that is commonly used in Python for instances like this. I'm offering up another way of doing this that would be familiar and readily come to mind for a long-time Python developer, but may not be immediately obvious or known to somebody less familiar with the language. :) 

As to timing, I wouldn't think that copy/pasting a suggestion would really have any impact on anything. But again, this is more an offer of another idiomatic approach than a necessary change or hard blocker.

Reply:
1 line is not always good. What happens when that line gets too long? Also, the current approach is easier to read. And finally, is it really worth this time?

Reply:
I don't believe @tarkatronic 's suggestion was focused on one line but rather on Pythonic approach, per PEP, to the dictionary.